models/games.model.js

```diff
@@ -1,7 +1,13 @@
- const db = require('../db/connection');
+const db = require("../db/connection");
 
- exports.selectAllCategories = () =>{
-   return db.query('SELECT * FROM categories;').then((categories)=>{
-        return categories.rows;
-   })
- } 
\ No newline at end of file
+exports.selectAllCategories = () => {
+  return db.query("SELECT * FROM categories;").then((categories) => {
+    return categories.rows;
+  });
+};
+
+exports.selectAllReviews = () => {
+  return db.query("SELECT * FROM reviews ORDER BY created_at DESC;").then((reviews) => {
```

Comment:
This needs to take into account the 'comment_count' property which is specified on the ticket.